components/link_tooltip/link_tooltip.tsx

```diff
@@ -12,7 +12,10 @@ const tooltipContainerStyles: CSSProperties = {
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
-    zIndex: 10,
+    zIndex: 1070,
```

Comment:
We should have a standardized zIndex value for all these zIndex layers :P.

Reply:
Hoshiyari nai ziada :p 
This is basically the z-index our bootstrap tooltips use right now.